apps/web/src/components/ui/password-form.tsx

```
@@ -0,0 +1,111 @@
+
+          <button
+            type="button"
+            tabIndex={-1}
+            disabled={disabled}
+            onClick={() => setShowPassword((prev) => !prev)}
+            className="flex h-4 w-4 flex-shrink-0 items-center justify-center disabled:cursor-not-allowed"
+          >
+            {showPassword ? (
+              <Eye
+                className={cn("h-4 w-4", disabled ? "text-placeholder" : "text-ink-black")}
+              />
```

ESLint

[error] Replace `⏎················className={cn("h-4·w-4",·disabled·?·"text-placeholder"·:·"text-ink-black")}⏎·············` with `·className={cn("h-4·w-4",·disabled·?·"text-placeholder"·:·"text-ink-black")}`

(prettier/prettier)